app.js

```diff
@@ -1,17 +1,54 @@
 const express = require('express');
-const {getTopics,getArticles} = require('./controllers');
-
+const {getTopics,getArticles,getArticleById,getCommentsArticle,getUsers,
+       postCommentByArticleId,
+       patchVote,
+} = require('./controllers');
 
 
 const app = express();
 
 app.use(express.json());
+
 // ==================   GET   ==================
        //  -----    3    ------
 app.get("/api/topics",getTopics);
 
+       //  -----    4    ------
+//🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸
```

Comment:
These emojis should be removed before your final commits on this repo. You may be showing this code to prospective employers.